src/components/WordSelector/WordSelector.scss

```diff
@@ -41,7 +41,7 @@
   color: $assignments-font-color;
 }
 
-.CloseButton {
+.CloseButtonClass {
```

Comment:
so are there like 2 classes now, one named CloseButton and the other one CloseButtonClass? thats kinda sus